client/src/modules/grades/templates/action.tmpl.html

```diff
@@ -0,0 +1,22 @@
+<div class="ui-grid-cell-contents text-center" uib-dropdown dropdown-append-to-body>
+  <a uib-dropdown-toggle href>
+    <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+    <span class="caret"></span>
+  </a>
+
+  <ul data-action="{{rowRenderIndex}}"  class="dropdown-menu-right" uib-dropdown-menu>
```

Comment:
Do you ever use `rowRenderIndex`? If not, it should be removed.

Reply:
It's used in E2E test